v22.1/security-reference/encryption.md

```diff
@@ -13,35 +13,27 @@ CockroachDB uses either TLS 1.2 or TLS 1.3 for inter-node and client-node [authe
 
 ## Encryption at rest
 
-Industry standard encryption-at-rest provided at the infrastructure level by your chosen infrastructure-as-a-service (IAAS) provider, either Google Cloud Platform (GCP) or Amazon Web Services (AWS). See documentation for [GCP persistent disk encryption](https://cloud.google.com/compute/docs/disks#pd_encryption) or [AWS Elastic Block Storage](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSEncryption.html).
+Industry standard encryption-at-rest is provided at the infrastructure level by your chosen infrastructure-as-a-service (IAAS) provider, either Google Cloud Platform (GCP) or Amazon Web Services (AWS). This helps to protect cloud resources from authorized access over the Information-As-A-Service (IAAS) platform's infrastructure. See documentation for [GCP persistent disk encryption](https://cloud.google.com/compute/docs/disks#pd_encryption) or [AWS Elastic Block Storage](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSEncryption.html).
```

Comment:
```suggestion
Industry standard encryption-at-rest is provided at the infrastructure level by your chosen cloud provider, either Google Cloud Platform (GCP) or Amazon Web Services (AWS). This helps to protect cloud resources from authorized access over the cloud provider's infrastructure. See documentation for [GCP persistent disk encryption](https://cloud.google.com/compute/docs/disks#pd_encryption) or [AWS Elastic Block Storage](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSEncryption.html).
```

Reply:
I'm going to leave the terminology as it was before I edited the topic. This is under discussion by the docs team and we will be sure to update terminology to keep it consistent. See https://cockroachlabs.atlassian.net/browse/DOC-4584.